docs/meta.md

```diff
@@ -12,60 +12,60 @@ This document describes how to define a blocklet that can be find/installed/mana
+|                                |          |           |                                                           |                                                                                                    | Draft  |
+| hooks                          | Object   | No        | Defines blocklet lifecycle scripts                        |                                                                                                    | Draft  |
+| hooks.pre_install              | String   | No        | Script to run before install the blocklet                 | Can use this to check environment requirements                                                     | Draft  |
+| hooks.pre_deploy               | String   | No        | Script to run before deploy the blocklet                  | Can use this to build blocklets                                                                    | Draft  |
+| hooks.pre_start                | String   | No        | Script to run before start the blocklet                   | Can use this to prepare some data for the blocklet                                                 | Draft  |
+| hooks.pre_stop                 | String   | No        | Script to run before stop the blocklet                    | Can use this to do some cleanup for the blocklet                                                   | Draft  |
+| hooks.pre_uninstall            | String   | No        | Script to run before uninstall the blocklet               | Can use this to do some cleanup for the blocklet                                                   | Draft  |
+|                                |          |           |                                                           |                                                                                                    | Draft  |
+| repository                     | Object   | No        | Defines blocklet source location                          |                                                                                                    | Draft  |
+| repository.type                | String   | No        | Repository type, can be `git`                             |                                                                                                    | Draft  |
+| repository.url                 | String   | No        | Repository URL                                            |                                                                                                    | Draft  |
+|                                |          |           |                                                           |                                                                                                    | Draft  |
+| charging                       | Object   | No        | A place holder for charging related                       |                                                                                                    | Draft  |
```

Comment:
charging --> paid

@robroyhobbs  can you also help to review those English words ?

Reply:
okay, no problem.